.gitleaks.toml

```diff
@@ -1,5 +1,5 @@
 [allowlist]
-  description = "Our test install exports a test only MINIO ACCESS KEY"
-  paths = [
+  files = [
     ".github/workflows/scripts/install.sh",
+    "docs/workflows/authentication.rst",
```

Comment:
I think in this specific case, we're better served with a `regexes = [ exact-key-here ]` entry for the not-a-secret itself. Esp since this file is in history-only and won't be modifying the key ever again...